akka-stream-tests/src/test/java/akka/stream/StreamTest.java

```diff
@@ -18,4 +21,14 @@ protected StreamTest(AkkaJUnitActorSystemResource actorSystemResource) {
         ActorMaterializerSettings settings = ActorMaterializerSettings.create(system);
         materializer = ActorMaterializer.create(settings, system);
     }
+
+    @Before
+    public void setUp() {
+        TestKit.stopAllStages(materializer);
```

Comment:
wouldn't it be better to create a new materializer?

I see that we do that automatically in the scaladsl, but somewhat weird to expose to end-users. I think they will find it more natural to create a new materializer

Reply:
I am not sure what do you mean by:

> somewhat weird to expose to end-users

Which part we should not expose? Materializer parameter to `stopAllStages`?

Reply:
My point is that stopAllStages is not needed. In docs we can show that new materializer is created in Before.